pkg/util/overridemanager/overridemanager.go

```diff
@@ -363,3 +460,27 @@ func parseJSONPatchesByPlaintext(overriders []policyv1alpha1.PlaintextOverrider)
 	}
 	return patches
 }
+
+func parseYAMLPatchesByField(overriders []policyv1alpha1.YAMLPatchOperation) []overrideOption {
+	patches := make([]overrideOption, 0, len(overriders))
+	for i := range overriders {
+		patches = append(patches, overrideOption{
+			Op:    string(overriders[i].Operator),
+			Path:  overriders[i].SubPath,
+			Value: overriders[i].Value,
+		})
+	}
+	return patches
+}
+
+func parseJSONPatchesByField(overriders []policyv1alpha1.JSONPatchOperation) []overrideOption {
+	patches := make([]overrideOption, 0, len(overriders))
+	for i := range overriders {
+		patches = append(patches, overrideOption{
+			Op:    string(overriders[i].Operator),
+			Path:  overriders[i].SubPath,
+			Value: overriders[i].Value,
```

Comment:
Just out of curiosity, what going to happen if the operation is `add` or `replace` but without `value`?

Reply:
The assigned value will be `null`.

Reply:
Should we add some validation for that? Something like:
- if the operation is `add` or `replace`, then the value must be set
- if the operation is `remove`, the value should be nil.

@chaunceyjiang any thoughts?

Reply:
good idea.

+1

Reply:
OK. I just added one more task to #5608.

@sophiefeifeifeiya Let me know if you'd like to take it.